src/Modules/Admin/Commands.Admin/PowerBIAdminClientCmdlet.cs

```diff
@@ -0,0 +1,57 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.PowerBI.Common.Api;
+using Microsoft.PowerBI.Common.Api.Encryption;
+using Microsoft.PowerBI.Common.Client;
+
+namespace Microsoft.PowerBI.Commands.Admin
+{
+    public abstract class PowerBIAdminClientCmdlet : PowerBIClientCmdlet
```

Comment:
The goal of this abstract class was to add more functionalities for Admin client if needed and not just for Encryption

Reply:
I'd like to discuss this further. We should carefully consider the names of these classes, especially since admin has existing meaning in the content of the Power BI API.

Reply:
Updated to PowerBIGetEncyptionKeyClientCmdlet